src/shared/userdb.c

```diff
@@ -469,6 +478,11 @@ static int userdb_start_query(
                 if ((flags & (USERDB_EXCLUDE_NSS|USERDB_AVOID_MULTIPLEXER)) && is_nss)
                         continue;
 
+                /* Similar for the drop-in service */
+                is_dropin = streq(de->d_name, "io.systemd.DropIn");
+                if ((flags & (USERDB_EXCLUDE_DROPIN|USERDB_AVOID_MULTIPLEXER)) && is_dropin)
```

Comment:
`FLAGS_SET` ?

Reply:
FLAGS_SET() checks if all specified bits are set. Here we want to check if either is set.